crates/pathfinder/src/consensus/inner/conv.rs

```diff
@@ -595,3 +598,536 @@ impl TryIntoDto<p2p_proto::common::L1DataAvailabilityMode> for u8 {
         Ok(res)
     }
 }
+
+impl IntoProto<FinalizedBlock> for dto::FinalizedBlock {
```

Comment:
I'm thinking that the trait name here, and for the other types that follow, is pretty unfortunate. Because the "proto" side of things are actually the `pathfinder_lib::common` types, as opposed to the types above, which are genuinely a representation of what goes onto the wire (ie. proposal parts). Maybe you could introduce a copy of the `IntoProto` trait but swap `[Pp]roto` for `[Cc]ommon`? To keep semantics correct.